traffic_ops/testing/api/v2/phys_locations_test.go

```diff
@@ -76,6 +79,33 @@ func GetTestPhysLocations(t *testing.T) {
 			t.Errorf("cannot GET PhysLocation by name: %v - %v", err, resp)
 		}
 	}
+
+}
+
+func GetSortPhysLocationsTest(t *testing.T) {
```

Comment:
What are your thoughts about adding these tests to the v1 tests also since it will change the ordering in v1 and v2? not sure its super important but might be worth doing